asv_bench/benchmarks/frame_ctor.py

```diff
@@ -118,4 +119,39 @@ def time_frame_from_range(self):
         self.df = DataFrame(self.data)
 
 
+class FromArrays:
+
+    goal_time = 0.2
+
+    def setup(self):
+        N_rows = 1000
+        N_cols = 1000
+        self.float_arrays = [np.random.randn(N_rows) for _ in range(N_cols)]
+        self.sparse_arrays = [
+            pd.arrays.SparseArray(np.random.randint(0, 2, N_rows), dtype="float64")
+            for _ in range(N_cols)
+        ]
+        self.int_arrays = [
+            pd.array(np.random.randint(1000, size=N_rows), dtype="Int64")
+            for _ in range(N_cols)
+        ]
+        self.index = pd.Index(range(N_rows))
+        self.columns = pd.Index(range(N_cols))
+
+    def time_frame_from_arrays_float(self):
+        self.df = DataFrame._from_arrays(
```

Comment:
I wouldn't change this if no other feedback but I don't think you need the assignment here in any of the benchmarks

Reply:
True (was only mimicking the other benchmarks in this file)

Reply:
hmm this is wrong, but yeah can clean this up in a followup (if you want to create a followup issue or PR to do it)